src/attack.c

```diff
@@ -156,22 +156,69 @@ set_scan_status (char *status)
 }
 
 /**
- * @brief Sends the status of a host's scan.
+ * @brief Send status to the client that the host is dead
+ *
+ * Originally the progress status is of the format
+ * "current_host/launched/total". Current host is the ip_str of the current host
+ * which is vulnerability tested. Launched is the number of plguins(VTs) which
+ * got already started. Total is the total number of plugins which will be
+ * started for the current host. But here we use the format "current_host/0/-1"
+ * for implicit singalling that the host ist dead.
+ *
+ * @param main_kb Kb to use
+ * @param hostnae ip_str representation of host
+ *
+ * @return 0 on success, -1 on failure.
+ */
+static int
+comm_send_status_host_dead (kb_t main_kb, char *hostname)
```

Comment:
I would use ip_str instead of hostname, since we are using the IP. Also, this change would fix the typo in the docstring.
```suggestion
comm_send_status_host_dead (kb_t main_kb, char *ip_str)
```